crazy_rl/multi_agent/jax/catch/catch.py

```diff
@@ -226,83 +180,49 @@ def state(self, state: State) -> jnp.ndarray:
 if __name__ == "__main__":
     from jax.lib import xla_bridge
 
-    jax.config.update("jax_platform_name", "gpu")
+    jax.config.update("jax_platform_name", "cpu")
 
     print(xla_bridge.get_backend().platform)
 
-    parallel_env = Catch(
-        num_drones=5,
+    num_agents = 5
+    env = Catch(
+        num_drones=num_agents,
         init_flying_pos=jnp.array([[0.0, 0.0, 1.0], [2.0, 1.0, 1.0], [0.0, 1.0, 1.0], [2.0, 2.0, 1.0], [1.0, 0.0, 1.0]]),
         init_target_location=jnp.array([1.0, 1.0, 2.5]),
         target_speed=0.1,
     )
 
-    num_envs = 1000  # number of states in parallel
+    num_envs = 3  # number of states in parallel
     seed = 5  # test value
     key = random.PRNGKey(seed)
-
-    vmapped_step = vmap(parallel_env.step_vmap)
-    vmapped_auto_reset = vmap(parallel_env.auto_reset)
-    vmapped_reset = vmap(parallel_env.reset)
-
-    @jit
-    def body(i, states_key):
-        """Body of the fori_loop of play.
-
-        Args:
-            i: number of the iteration.
-            states_key: a tuple containing states and key.
-        """
-        states, key = states_key
-
-        key, *subkeys = random.split(key, parallel_env.num_drones + 1)
+    key, *subkeys = random.split(key, num_envs + 1)
+
+    # Wrappers
+    env = NormalizeObservation(env)
```

Comment:
I think we shouldn't show all the wrappers here. VecEnv and AutoReset maybe but not the others.